custodian/vasp/jobs.py

```diff
@@ -673,7 +673,18 @@ def terminate(self):
         """
         logger.info(f"Custodian terminating all VASP processes within process group {self.sbprcss.pid}")
         try:
-            os.killpg(os.getpgid(self.sbprcss.pid), signal.SIGTERM)
+            try:  # first try to terminate all VASP processes based on sbprcss.pid
```

Comment:
I don't think we need 2 levels of `try` here. There's an unused except case below.

Reply:
I agree—I was just matching the unused `except` from the original code (which was present both before and after #231 )